Resources/Prototypes/Entities/Mobs/NPCs/animals.yml

```diff
@@ -3377,6 +3377,11 @@
   - type: NpcFactionMember
     factions:
     - Passive
+  - type: HTN
+    rootTask:
+      task: RuminantCompound
+  - type: Body
+    prototype: AnimalHemocyanin
```

Comment:
```suggestion
```
You also changed pigs to have crab organs. Is that [carcinisation](https://en.wikipedia.org/wiki/Carcinisation) at work?

Reply:
I have removed this error, pigs no longer have claws.